js/tabulus.js

```diff
@@ -13,7 +13,19 @@ function Tabulus() {
     ** Main Function Object
     */
     function my(sel) {
-      if(!container) setup(sel);
+        if(!container) setup(sel);
+        d3.selectAll("a[data-toggle='modal']")
+            .each(function() {
+                var self = d3.select(this)
+                  , name = self.attr("data-name")
+                ;
+                self
```

Comment:
I see. Thanks for clarifying. Those seem like invalid values, right? Could we just remove those `href` and `data-target` from the Jekyll template?